dxr/app.py

```diff
@@ -307,6 +307,14 @@ def browse(tree, path=''):
                             frozen['generated_date'])
 
 
+def concat_plugin_headers(plugin_list):
```

Comment:
I was going to say something about ["nouning" this procedure name](https://grinchcentral.com/function-names-to-verb-or-not-to-verb), but it's already a mess in app.py, so there's no clear convention. :-(